tests/e2e/travis-e2e.sh

```diff
@@ -8,12 +8,14 @@ set -u
 # print each command before executing it
 set -x
 
+K8S_VERSION="${K8S_VERSION:-"1.19.0"}"
+
 curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
 chmod +x kubectl
-curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64
+curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/v0.8.1/kind-linux-amd64
```

Comment:
As 1.19 was released after kind v0.8.1, I think we should bump to the more recent v0.9.0 instead.

Reply:
Makes sense. This PR was created before kind v0.9.0 was released :)